config/config.ini.example

```diff
@@ -60,86 +41,44 @@
 
 #db-cleanup                     # Enable regular database cleanup thread.
-#db-cleanup-worker              # Clear worker status from database after X minutes of inactivity. Default: 30, 0 to disable.
 #db-cleanup-pokemon             # Clear pokemon from database X hours after they disappeared. Default: 0, 0 to disable.
 #db-cleanup-gym                 # Clear gym details from database X hours after last gym scan. Default: 8, 0 to disable.
 #db-cleanup-spawnpoint          # Clear spawnpoint from database X hours after last valid scan. Default: 720, 0 to disable.
 #db-cleanup-forts               # Clear gyms and pokestops from database X hours after last valid scan. Default: 0, 0 to disable.
 
 
-# Scan method (speed-scan preferable, (default is hex-scan)
-#############
-
-#beehive                        # Use beehive configuration for multiple accounts, one account per hex.
-                                # Make sure to keep -st under 5, and -w under the total amount of accounts available.
-#spawnpoint-scanning            # Use spawnpoint scanning (instead of hex grid).
-                                # Scans in a circle based on step_limit when on DB.
-                                # A spawnpoint-file can be added optinal as value for "spawnpoint-scanning".
-#dump-spawnpoints               # Dump the spawnpoints from the db to json (only for use with -ss).
-#ss-cluster-time                # Time threshold in seconds for spawn point clustering (0 to disable). (only for use with -ss).
-
-
-# Geofence settings
-###################
-
-#geofence-file:                 # Geofence file to define outer borders of the scan area.
-#geofence-excluded-file:        # File to define forbidden areas inside scan area. Regarded this as inverted geofence. Can be combined with geofence-file.
-
-
-
-# Pokemon IV (Only use one of the filters below)
+# Pokemon IV
 ############
 
 #encounter                      # Set to true to start encounters to pull more info, like IVs or movesets. (default=False)
-#encounter-delay:               # Delay in seconds before starting an encounter. Must not be zero. (default=1)
-#high-lvl-accounts:             # File containing a list high level accounts, in the format "auth_service,username,password"
-#enc-whitelist-file:            # File containing a list of Pokemon IDs to encounter for IV/CPs. Requires L30 or higher accounts in --high-lvl-accounts.
-#hlvl-kph:                      # Set a maximum speed in km/hour for high level account scanning. 0 to disable. (default=25)
-#medalpokemon                   # Show notify for tiny rattata and big magikarp on map
+#medalpokemon                   # Notify of tiny rattata and big magikarp on map. (default=False)
 
 # Webserver settings
 ####################
 
-#only-server                    # Server-Only Mode
 #locale:                        # Pokemon translation
-#no-fixed-location              # Disables the fixed map location and shows the search bar for use in shared maps.
```

Comment:
So now we know what it's used for, it could be added back? :upside_down_face:

Reply:
😀

Reply:
It's back.